src/index.html

```diff
@@ -11,7 +11,138 @@
+      method="post"
+      name="test_form">
+
+      <fieldset name="personal_information" class="field">
+        <legend>Personal information</legend>
+        <label class="box">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="box">Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="box">
+          How old are You?
+          <input
+            type="number"
+            name="year"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="box">
+          Full date of birth:
+          <input type="date" name="date">
+        </label>
+        <label class="box">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset name="registration" class="field">
+        <legend>Registration</legend>
+        <label class="box">E-mail:
+          <input
+            type="email"
+            placeholder="email@example.com"
+            name="e-mail"
+          >
+        </label>
+        <label class="box">Password:
+          <input
+            type="password"
+            minlength="8"
+            name="pass"
+          >
+        </label>
+      </fieldset>
+      <fieldset name="facts" class="field">
+        <legend>An interesting fact about you!</legend>
+        <label class="box">Do you love cats?
+          <label class="radio">
+            <input
+              type="radio"
+              value="yes"
+              name="cats"
+            >
+            Yes
+          </label>
+          <label class="radio">
+            <input
+              type="radio"
+              value="no"
+              name="cats"
+            >
+            No
+          </label>
+        </label>
+        <label class="box">What is your favorite color?
+          <input type="color" name="color">
+        </label>
+        <label class="box">What time do you go to bed?
+          <input type="time" name="time">
+        </label>
+        <label class="box">What are your favorite brand of cars?
+          <select
+            name="cars"
+            size="4"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+        <label class="box">How do you rate our work?
+          <input
+            type="range"
+            name="rate"
+            value="0"
+            min="0"
+            max="10"
+          >
+        </label>
+      </fieldset>
+      <fieldset name="additional_info" class="field">
```

Comment:
```suggestion
      </fieldset>

      <fieldset name="additional_info" class="field">
```

Add empty line between multiple sibling blocks